docs-src/typescript/clients.md

```diff
@@ -41,13 +41,15 @@ import { example } from './workflows';
 
 async function run() {
   // Connect to the default Server location (localhost:7233)
+  console.log(Date.now());
```

Comment:
Third PR where this same auto-sync'd stuff is coming in. Maybe you need another PR that puts this stuff in main instead of each of these?